File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
   public int[] twoSum(int[] nums, int target) {

        int[] resenje = new int[2];

        int i=0;
        while (i<nums.length) {
            for (int k=i+1;k<nums.length;k++) {
                if (nums[i]+nums[k]-target==0) {
                    resenje[0]=i;
                    resenje[1]=k;
                    break;
                }

        }
            i++;
        }
        return resenje;
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        resenje = nums.copy()
        i =0
        while i<len(nums):
            proizvod =1

            for k in range(len(nums)):
                if k!=i:
                    proizvod*=nums[k]
            resenje[i]=proizvod
            i+=1

        return resenje